scripts/brp-python-bytecompile

```diff
@@ -87,6 +87,10 @@ if [ ! -x "$default_python" ]; then
 	exit 0
 fi
 
+# Figure out if there are files to be bytecompiled with the default_python at all
+# this prevents unnecessary default_python invocation
+find "$RPM_BUILD_ROOT" -type f -name "*.py" | grep -Ev "/bin/|/sbin/|/usr/lib(64)?/python[0-9]\.[0-9]|/usr/share/doc" || exit 0
```

Comment:
Note that the regex violates the DRY principle. However it was already repeated twice and I didn't want to add an extra layer of complexity by putting it in a variable instead. However I can do that if you'd like.